python/tests/integration/arcticdb/test_lmdb.py

```diff
@@ -149,3 +154,32 @@ def test_lmdb_options_unknown_option(options):
         la.parse_query(options)
 
     assert "Invalid LMDB URI" in str(e.value)
+
+
+def create_arctic_instance(tmpdir):
+    ac = Arctic(f"lmdb://{tmpdir}")
+
+
+def test_warnings_arctic_instance(tmpdir):
+    ac = Arctic(f"lmdb://{tmpdir}")
+    # should warn
+    ac = Arctic(f"lmdb://{tmpdir}")
+
+    del ac
+    # should not warn
```

Comment:
Checking the warning is actually produced is carried out manually then?

Reply:
Yeah, checking the logging was super difficult and didn't seem worth it